src/Widgets/NotificationEntry.vala

```diff
@@ -78,17 +80,41 @@ public class Notifications.NotificationEntry : Gtk.ListBoxRow {
             column_spacing = 6,
             margin = 12,
             // Box shadow is clipped to the margin area
-            margin_top = 1,
+            margin_top = 14,
             margin_bottom = 11
         };
 
         unowned Gtk.StyleContext grid_context = grid.get_style_context ();
         grid_context.add_class (Granite.STYLE_CLASS_CARD);
         grid_context.add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
 
+        var delete_image = new Gtk.Image.from_icon_name ("window-close-symbolic", Gtk.IconSize.SMALL_TOOLBAR);
+        delete_image.get_style_context ().add_class ("close-image");
```

Comment:
As above, adding this close is not necessary
```suggestion
```

Reply:
See above for explanation as to why I did it. Something blocks me from coloring the image if I don't do this.

Reply:
Suddenly, it worked now.